deploy-queue/action.yml

```diff
@@ -172,3 +172,47 @@ runs:
             exit 1
             ;;
         esac
+
+    - name: Send start notification to Slack
+      id: send-slack-start
+      if: steps.run-deploy-queue.outcome == 'success' && inputs.mode == 'start' && inputs.environment == 'prod'
+      uses: slackapi/slack-github-action@e28cf165c92ffef168d23c5c9000cffc8a25e117 # v1.24.0
+      with:
+        channel-id: 'C09QDUK37QW' # channel lakebase-deploy-queue-stream
```

Comment:
I believe this should be an optional input for the action (if not set, then we don't need to post the message)